README.markdown

```diff
@@ -150,7 +150,11 @@ Specifies a file mode for the ntp configuration file. Valid options: string cont
 
 ####`config_template`
 
-Specifies a file to act as a template for the config file. Valid options: string containing a path (absolute, or relative to the module path). Default value: 'ntp/ntp.conf.epp'
+Specifies a file to act as a ERB template for the config file. Valid options: string containing a path (absolute, or relative to the module path). Default value: 'ntp/ntp.conf.erb'. Validation error will be thrown if this param is supplied as well as the `config_epp` param.
```

Comment:
The default here is not right. Maybe say "example value", so that people have something to grab onto.